.changelog/16811.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+driver/docker: Default `devices.container_path` to `devices.host_path` like Docker's CLI.
```

Comment:
No need for the trailing `.`

Using `make cl` to generate these will help format them